webapp/src/constants.ts

```diff
@@ -56,7 +56,7 @@ class Constants {
         {
             code: 'ru',
             name: 'russian',
-            displayName: 'Pусский (Beta)',
```

Comment:
All this (Beta) labels are about the translation to that language, not about the stage of the project, I think this labels doesn't need to be changed.

Reply:
Apple doesn't like seeing "Beta" in the UI. Could we rename this, e.g. to "level 1" or "partial"?

Reply:
That is an usability problem in my opinion. Even removing the Beta entirely and not replacing that is a matter of user expectations, so from the technical side I only can say that I don't love it to just change to a language and feel like is partially translated by surprise.

@asaadmahmood what do you think? should we change this text to something like `(Partial)` or `(In Progress)` or `(Incomplete)` or something like that? do you think we should just remove it entirely?

Reply:
@jespino I don't understand the exact problem. Is the reason we're removing this in the first place because of Marketing?
Renaming it to level 1, partial, in progress/incomplete, doesn't make sense to me.

What exactly do we want to tell the user by this label, and where does it appear?

Reply:
The issue is we cannot show "Beta" in the final UI for the Mac App Store (it will fail App review). I can see two ways to address this:
1. Change the label in Weblate to "partial" (or similar)
2. Write code to strip out the "Beta" and "Alpha" labels at runtime

I suppose a third option is to omit all the Beta / Alpha languages.

Do you see another alternative?

Reply:
@chenilim I'm fine then hiding the beta tag, I am fine using the month for the release like we talked about.

Reply:
Thanks @asaadmahmood! @jespino, do you see any other issues before I merge this? Thanks.